bakery/env/local.json

```diff
@@ -1,7 +1,7 @@
 {
   "ENV_NAME": "local",
   "COPS_TARGET": "http://backend/api",
-  "COPS_ARTIFACTS_S3_BUCKET": "",
+  "COPS_ARTIFACTS_S3_BUCKET": "ce-thomas-sandbox",
```

Comment:
This looks like an inadvertent change in your recent commit.

Reply:
Lol i was very careful adding files, then i did `commit -a` like a dunce.

Reply:
fixed